public/styles.css

```diff
@@ -1,3 +1,61 @@
-body {
-  margin: 0;
+#main {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+}
+.mainBox {
+  flex-direction: column;
+  /*align-items: center;*/
```

Comment:
a co to za zakomentowany kod w kodzie produkcyjnym ?! :P tego typu rzeczy sugerują, że nie posprzątałeś za sobą. Kod nadmiarowy to kod do wyrzucenia :D